coalib/bearlib/aspects/Formatting.py

```diff
@@ -0,0 +1,393 @@
+from coalib.bearlib.aspects import Root, Taste
+
+
+@Root.subaspect
+class Formatting:
+    """
+    Formatting refers to the visual appearance of you source code.
+    """
+    class docs:
+        example = """
+        # Here is an example of python code with lots of
+        # formatting issues including: trailing spaces, missing spaces
+        # around operators, incorrect indentation etc...
+
+        z = 'hello '+'world'
+        .....def f  (   a ) :
```

Comment:
no they dont.  `...` is valid syntax in Python 3, called an ellipsis.  https://en.wikipedia.org/wiki/Ellipsis_(computer_programming)

what syntax is this docstring being encoded in? rst?  use rst code blocks or rst literals